docs/ansible/roles/debops.borgbackup/inventory/host_vars/backup.example.net/default.yml

```diff
@@ -0,0 +1,13 @@
+---
+# vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# cron [[[
+
+cron__host_jobs:
+
+  'backup':
+    job: '/usr/local/bin/borgmatic --create --stats >> /var/log/backup.log 2>&1'
```

Comment:
I think the path should be dynamic, ether if it is install through package manager or though pip

Reply:
This will resolve itself after buster. Does that work for you?